cmd/fleet/serve.go

```diff
@@ -180,6 +191,10 @@ the way that the Fleet server works.
 				os.Exit(1)
 			}
 
+			if config.Auth.JwtKey != "" && config.Auth.JwtKeyPath != "" {
```

Comment:
This check needs to go above the loading of the key from the file -- otherwise I think it's going to always trigger if the path is specified.